src/features/DocumentFormatter.ts

```diff
@@ -139,6 +139,7 @@ class PSDocumentFormattingEditProvider implements DocumentFormattingEditProvider
     private readonly ruleOrder: string[] = [
         "PSPlaceCloseBrace",
         "PSPlaceOpenBrace",
+        "PSUseWhitespace",
```

Comment:
I was in a bit of a limbo on whether to have "consistent" in its name. But looks like "PSUseConsistentWhitespace" sounds more consistent with the "PSUseConsistentIndentation" rule. On the other hand all the rules enforce some or the other kind of consistency and so having the "consistent" word seemed verbose (And I was even thinking of changing "PSUseConsistentIndentation" to "PSUseIndentation", but that would be a breaking change for PSSA). At this point, I personally don't have any preference but if you think "PSUseConsistentWhitespace" sounds better, I can change it to that.